lms/lmsweb/tools/registration.py

```diff
@@ -1,16 +1,54 @@
 import csv
+from lms.lmsweb.tools.validators import (
+    UniqueEmailRequired, UniqueUsernameRequired,
+)
 import os
 import typing
 
+from flask import url_for
 from flask_babel import gettext as _  # type: ignore
+from flask_mail import Message  # type: ignore
+from flask_wtf import FlaskForm
+from itsdangerous import URLSafeTimedSerializer
+from wtforms import PasswordField, StringField
+from wtforms.validators import Email, EqualTo, InputRequired, Length
 
 from lms.lmsdb import models
-from lms.lmsweb import config
+from lms.lmsweb import config, webapp, webmail
 from lms.utils.log import log
 
 import requests
 
 
+SERIALIZER = URLSafeTimedSerializer(config.SECRET_KEY)
+
+
+class RegisterForm(FlaskForm):
```

Comment:
Maybe we better move this one to `lmsweb/forms/register.py`?

Reply:
I'd put in into `lmsweb/forms.py`, so for the future we'll have a place to put `forgot my password` and `change password` forms